llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp

```diff
@@ -666,7 +666,6 @@ bool LoopVectorizationLegality::canVectorizeOuterLoop() {
   // Check whether we are able to set up outer loop induction.
   if (!setupOuterLoopInductions()) {
     reportVectorizationFailure("Unsupported outer loop Phi(s)",
-                               "Unsupported outer loop Phi(s)",
```

Comment:
Small thought - do we want to unify the style for all `reportVectorizationFailure` calls to either capitalize or not-capitalize the first word of the string? As currently it looks like it's a mix between both.

Otherwise LGTM, once clang-format is happy.

Reply:
Yeah it's a good point and I thought about that too. It doesn't look like there is always consistency throughout for all of the reports. I agree it would be good to standardise them. Not sure if @fhahn has any thoughts?